app/main.py

```diff
@@ -1,3 +1,6 @@
 def count_occurrences(phrase: str, letter: str) -> int:
-    # write your code here
-    pass
+    counter = 0
+    for char in phrase:
+        if char.lower() == letter.lower():
```

Comment:
Think about do you need every time a for loop do `letter.lower()` if it's always the same?

Also, you can solve this task via some online methods, try to figure it out, or google it!